plugins/org.jkiss.dbeaver.data.office/OSGI-INF/l10n/bundle_pt_BR.properties

```diff
@@ -28,4 +26,4 @@ dataTransfer.processor.xlsx.property.splitByRowCount.description=Dividir por con
 dataTransfer.processor.xlsx.property.splitByColNum.name=Coluna do grupo
 dataTransfer.processor.xlsx.property.splitByColNum.description=N\u00famero da coluna para agrupar linhas na folha por valor da coluna
 dataTransfer.processor.xlsx.property.dateFormat.name=Formato de dados excel
-dataTransfer.processor.xlsx.property.dateFormat.description=Formato de excel e datas (e.x. m/d/yy hh:mm) vo�e pode mudar no excel.
\ No newline at end of file
+dataTransfer.processor.xlsx.property.dateFormat.description=Formato de excel e datas (e.x. m/d/yy hh:mm) vo�e pode mudar no excel.
```

Comment:
This and mentioned by Asya line seems accidentally modified. Please, return it back.